### ISSUE TITLE
Explorer folding and moving randomly when opening a file

### ISSUE THREAD

**Issue by toxinu** [microsoft/vscode#104613]
- VSCode Version:1.48.0
- OS Version:Darwin x64 19.6.0

Version: 1.48.0
Commit: db40434f562994116e5b21c24015a2e40b2504e6
Date: 2020-08-13T08:03:27.059Z (18 hrs ago)
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0

Steps to Reproduce:

1. Create a directory structure like that:

```bash
$ tree -a .
.
├── .vscode
│   └── settings.json
├── folder-01
│   ├── another-folder-01
│   │   └── file-01.md
│   ├── another-folder-02
│   │   └── file-01.md
│   ├── another-folder-03
│   │   └── file-01.md
│   ├── file-01.md
│   └── file-02.md
├── folder-02
│   ├── file-01.md
│   └── file-02.md
└── workspace.code-workspace

6 directories, 9 files
```

With `.vscode/settings.json`:

```json
{
  "files.exclude": {
      "folder-01/": true,
      "folder-02/": true,
  },
}
```

And `workspace.code-workspace` like:

```json
{
  "folders": [
    { "path": "folder-01" },
    { "path": "folder-02" },
    {
        "path": ".",
	"name": "project root"
    }
  ],
}
```

2. Open the workspace with `code workspace.code-workspace`
3. Clicking randomly on files and you should see the explorer jumping, folding, unfolding everything randomly
<img width="400px" src="https://user-images.githubusercontent.com/293337/90209819-4c18b580-de27-11ea-842f-0c543a837026.gif">

4. It really looks like the combination of `files.exlude` + 
```
{
    "path": ".",
    "name": "project root"
}
```
is driving VSCode like crazy.

Does this issue occur when all extensions are disabled?: **Yes**/~No~

**Comment by isidorn** [comment]
Thanks for filling this issue. Can you reproduce using Visual Studio Code Insiders https://code.visualstudio.com/insiders/

**Comment by toxinu** [comment]
@isidorn Thanks for your reply. And yes I just tested and I can reproduce it with the Insiders build.

```
Version: 1.49.0-insider
Commit: bd08768fd310d34cdd4423a53e5f3ca77340fea5
Date: 2020-08-14T06:47:57.598Z (1 hr ago)
Electron: 9.2.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
```

**Comment by vennekilde** [comment]
Sounds like a duplicate of https://github.com/microsoft/vscode/issues/95056
Only fix i know right now is to downgrade to < 1.44
